### PR TITLE
176507970 tray content and collection

### PR DESCRIPTION
**Pull request by mklewandowski**
This PR adds leaves to the tray and adds the ability to select a tray animal and click a notebook slot to potentially categorize the animal.

Changes include:
- add leaf assets
- add leaf exports to sim-utils
- add leaves to sorting tray (for now we will only add a small number and place them on the bottom, this will change once we have more nuanced sorting)
- clicking an animal in the sorting tray will highlight the animal (highlight is temporary until we have more assets), user can then click a slot in the notebook to attempt to match the animal.  If the match is correct, the animal is removed from the tray and added to the notebook.
- add React modal library
- add modal dialog that is presented when incorrect tray/notebook match occurs
- add translated text

collect

### CODE REVIEW

**Review by eireland — CHANGES_REQUESTED** [review]
The repo had `data-testid` for element selectors for the tests. Suggested switching the`data-cy`s to `data-testid` for uniformity

File: src/components/modal-dialog.tsx [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+      onClose();
+    };
+    return (
+      <Modal
+        isOpen={showModal}
+        contentLabel={title || "Alert"}
+        className={"modal-dialog"}
+      >
+        <div className="header" data-cy="modal-dialog-header">{title || "Alert"}</div>
+        <div data-cy="modal-dialog-label">{label}</div>
```
[the remark this answers is not in the document]
Comment (mklewandowski) [reply src/components/modal-dialog.tsx]: Fixed.

File: src/components/modal-dialog.test.tsx [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+describe("Modal Dialog component", () => {
+  it("renders Modal Dialog component", () => {
+    const wrapper = shallow(<ModalDialog
+                              title="test"
+                              label="dialog test"
+                              showModal={true}
+                              onClose={() => { /* nop */ }}
+                            />);
+    expect(wrapper.find('[data-cy="modal-dialog-header"]').length).toBe(1);
```
[the remark this answers is not in the document]
Comment (mklewandowski) [reply src/components/modal-dialog.test.tsx]: I removed this test for now.  It came from the AP - I don't actually want enzyme tests, but want to use react testing library instead.

File: src/components/modal-dialog.tsx [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+        <div className="footer">
+          <button onClick={handleClose} data-cy="modal-dialog-close">Close</button>
```
[the remark this answers is not in the document]
Comment (mklewandowski) [reply src/components/modal-dialog.tsx]: Fixed.

File: src/components/modal-dialog.tsx [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+    const handleClose = () => {
+      onClose();
+    };
+    return (
+      <Modal
+        isOpen={showModal}
+        contentLabel={title || "Alert"}
+        className={"modal-dialog"}
+      >
+        <div className="header" data-cy="modal-dialog-header">{title || "Alert"}</div>
```
[the remark this answers is not in the document]
Comment (mklewandowski) [reply src/components/modal-dialog.tsx]: Fixed.

**Review by eireland — APPROVED** [review]
:+1:

**Pull request merged**